tabbycat/api/serializers.py

```diff
@@ -141,6 +141,7 @@ class V1LinksSerializer(serializers.Serializer):
         institutions = serializers.HyperlinkedIdentityField(view_name='api-global-institution-list')
 
     _links = V1LinksSerializer(source='*', read_only=True)
+    site_timezone = serializers.CharField(allow_blank=False, trim_whitespace=True, read_only=True)
```

Comment:
How about name it just `timezone`? Also, why use `trim_whitespace=True`? It's also only used for schema generation...